src/translations/qjackctl_fr.ts

```diff
@@ -567,12 +567,12 @@ de toutes les applications clientes.
     <message>
         <location filename="../qjackctlGraphForm.ui" line="199"/>
         <source>Ctrl+C</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+C</translation>
```

Comment:
There's no need to translate this, as the translation system will automatically take the English version when compiling. What's more, there's a risk of errors or omissions in the long term (especially if the English string is changed).

Reply:
Well then we should review all the other shortcuts which have been translated.
Moreover, the key is currently the same as the source version but is actually determined by the "&<letter>" syntax in the menu text translation string.
A better fix would be to grab this programmatically and not rely on a separate string.

Reply:
> Well then we should review all the other shortcuts which have been translated.

Agreed.

Reply:
I've removed the translations where appropriate.

Reply:
Thanks for that.